pennylane/transforms/mitigate.py

```diff
@@ -450,9 +466,9 @@ def mitigate_with_zne(
             scale_factors=[1., 2., 3.],
             folding=fold_global,
             extrapolate=poly_extrapolate,
-            extrapolate_kwargs={'order': 2})
+        )
```

Comment:
I get this error when running the example:
`TypeError: mitigate_with_zne() missing 1 required positional argument: 'extrapolate'`

Reply:
Should be good now.